DeepCat.js

```diff
@@ -329,4 +331,6 @@
 			} );
 		} );
 	}
+
+	window.DeepCat = DeepCat;
```

Comment:
Common practice is to make this accessible from the outside via `mw.libs.deepCat`.